### ISSUE TITLE
Formspec model[] element documentation is very unclear.

### ISSUE THREAD

**Issue by FatalError42O** [minetest/minetest#14960]
### Minetest version

```
5.8.0
```

### Operating system and version

Windows 10

### Summary

I am currently working on automatically a side view of models and I'm finding some scales cause models of varying sizes to disappear randomly. the model scales proportionally somehow on both axises, but it also seems to determine an aspect ratio. I currently have the bounding boxes for every model calculated and it is still impossible to calculate the width and size of the element for every model seemingly, as I do not know the exact function of height and width.

It also isn't known whether it moves the model to fit into the image. The documentation needs to be more specific, as this is currently only usable with trial and error and not algorithmically.

### Steps to reproduce

Try using any models in a model formspec.

**Comment by FatalError42O** [comment]
for reference: this is my code for a formspec displaying a rendering of a gun. It only works for some weapons.
![image](https://github.com/user-attachments/assets/137ce42e-b3a0-4c6e-a16e-c9ebd45ba27f)

![image](https://github.com/user-attachments/assets/0b43eb84-777f-4fd0-b4df-cbcdaee538ea)
![image](https://github.com/user-attachments/assets/afdacf83-ee9d-411c-adf5-1922e4212b04)
![image](https://github.com/user-attachments/assets/f668fc8d-ec77-4b5d-ab2f-c12c6719bfec)
![image](https://github.com/user-attachments/assets/85753b9d-dfb2-402f-ad52-32726ffc9573)

the bounding boxes are confirmed to be correct, the behavior is not clear at all especially when compared to other elements like the box[].

**Comment by appgurueu** [comment]
I believe the way you're supplying rotation is incorrect: You need to provide it as `x,y`, not `{x,y}`. Since we use `stof` internally (see `GUIFormSpecMenu::parseModel`), the latter will lead to rotation along both axes being 0.

I assume `rotation {X,Y}` in the docs led you to this? I agree that's a bit confusing, especially since it is formatted as code. We should probably remove the curly braces there.

**Comment by FatalError42O** [comment]
@appgurueu this is an unrelated issue!!! the problem is with the height and the width not having a clearly defined meaning, the rotation thing is a totally unrelated nitpick!

**Comment by appgurueu** [comment]
I see. Relevant code is scattered all over https://github.com/minetest/minetest/blob/master/src/gui/guiScene.cpp, have a blast. Your issue is about more than just documentation though?

> I'm finding some scales cause models of varying sizes to disappear randomly.

This sounds like a bug with culling.

**Comment by FatalError42O** [comment]
> I see. Relevant code is scattered all over https://github.com/minetest/minetest/blob/master/src/gui/guiScene.cpp, have a blast. Your issue is about more than just documentation though?
> 
> > I'm finding some scales cause models of varying sizes to disappear randomly.
> 
> This sounds like a bug with culling.

yeah initially it was just an issue about the docs not telling you clearly what it meant, but there's also a clipping thing which is kind of it's own issue
